install

```diff
@@ -93,8 +93,8 @@ install() {
   echo "$FPREFIX Installing Swarmlet"
 
   echo "$PREFIX Installing required packages"
-  $SUDO "$PKG_MANAGER" update -y -qq &>/dev/null
-  $SUDO "$PKG_MANAGER" install -y -qq git apt-transport-https &>/dev/null
+  $SUDO "$PKG_MANAGER" update -y
```

Comment:
Can we wrap this in an `if` statement?
Something like:
```sh
if [ $PKG_MANAGER = 'deb' ]; then
  $SUDO "$PKG_MANAGER" update -y
  # ...
else
  $SUDO "$PKG_MANAGER" update -y -qq &>/dev/null
  # ..
fi
```
I'm not sure if this will break the headless installation.

Reply:
> Can we wrap this in an `if` statement?

actually I don't see the point of a `if` statement...

Reply:
> Reason: I'm not sure if this will break the headless installation.

It won't :) See: https://github.com/n-rodriguez/swarmlet/actions